### ISSUE TITLE
Lots of Sublime Text file type icons

### ISSUE THREAD

**Issue by ghost** [ivogabe/Brackets-Icons#52]
OK. You might wonder, "why doesn't he just create his extension?". Well I could, but, I don't want to override your stuff.
***
LaTeX
=====
Can you add:
```js
addIcon('tex', 'ion-document-text', '#960000');
addAlias('bib', 'tex');
addAlias('sty', 'tex');
```
Singular Types
============
Can you add:
```js
addIcon('applescript', 'ion-social-apple', '#afafaf');
addIcon('textile', 'ion-quote', '#960000');
addIcon('matlab', 'ion-clipboard', '#960000');
addIcon('lisp', 'ion-ios-paperplane', '#960000');
addIcon('xsl', 'ion-code', '#960000');
addIcon('tcl', 'ion-code', '#960000');
addIcon('rst', 'ion-ios-paper', '#960000');
addIcon('re', 'ion-ios-barcode-outline', '#960000');
addIcon('d', 'ion-contrast', '#960000');
addIcon('r', 'ion-ios-analytics', '#960000');
```

**Comment by ivogabe** [comment]
Do you see a chance creating a pull request? Or shall I add them?

**Comment by timbuktuuu** [comment]
Look at PR #66

**Comment by ivogabe** [comment]
Merged #66
